### DISCUSSION TITLE
在vercel中手动输入构建和安装指令进行部署，在运行启动代码时总是报错

### DISCUSSION THREAD

**Discussion by iliaoke** [GaiZhenbiao/ChuanhuChatGPT#940] [Q&A]
### 是否已存在现有反馈与解答？

- [X] 我确认没有已有issue或discussion，且已阅读**常见问题**。

### 错误描述

vercel部署时发生的错误

### 复现操作

无

### 错误日志

```shell
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
  File "ChuanhuChatbot.py", line 312
    placeholder=f"word:likelihood",
                                 ^
SyntaxError: invalid syntax
Error: Command "python ChuanhuChatbot.py" exited with 1
```

### 运行环境

- OS: 
- Browser: 
- Gradio version: 
- Python version: 
(或您的其他运行环境信息)

**Comment by Keldos-Li** [comment]
不是很清楚vercel部署会怎么样。如果vercel使用Python，也请确保使用Python3.9及以上版本。

**Comment by iliaoke** [comment]
输入命令检测了一下是Python版本太低了。但官方说默认的Python版本是3.9。我实际检测是python2，网上没查到怎么在vercel中自定义Python版本。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月2日(周一) 下午3:18
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [GaiZhenbiao/ChuanhuChatGPT] 在vercel中手动输入构建和安装指令进行部署，在运行启动代码时总是报错 (Issue #904)




 
不是很清楚vercel部署会怎么样。如果vercel使用Python，也请确保使用Python3.9及以上版本。
